src/Microsoft.Identity.Client/Platforms/Android/WebUIFactory.cs

```diff
@@ -28,7 +28,7 @@
 using Microsoft.Identity.Client.Internal;
 using Microsoft.Identity.Client.Internal.Interfaces;
 
-namespace Microsoft.Identity.Client
+namespace Microsoft.Identity.Client.Internal.WebUIs
```

Comment:
it should be WebUI.

Reply:
Then we'd have a possible naming confict with the WebUI class, which obliges to make the code more complex by adding a type using (using Microsoft.Identity.Client.Internal.WebUI.WebUI) ?

Reply:
This would not be a conflict. Actually we could just use a more generic name like UI. in case we ever have something like webUI and nativeUI (think of iOS/Android providing native controls to collect creds).